### PR TITLE
Qingwei/faster user input

### PR DESCRIPTION
**Pull request by qingweibinary**

### CODE REVIEW

File: www/index.html [hunk 1]
```diff
@@ -67,7 +67,7 @@
             },
             onDeviceReady: function() {
                 document.addEventListener("backbutton", onBackPressed, false);
-                codePush.sync();
+//                codePush.sync();
```
Comment (borisyankov) [inline www/index.html]: Hmmm?

Comment (qingweibinary) [inline www/index.html]: Want to suggest we disable it at the moment? seems not reliable, and we have not verify it, it caused problem with QA as they keep test on an outdated version despite you've published

Comment (borisyankov) [inline www/index.html]: The moment we publish a line with it, this version becomes non auto-updatable.

Comment (borisyankov) [inline www/index.html]: We better debug that as a first priority.

Comment (qingweibinary) [inline www/index.html]: Added it back

**Pull request merged**